misc/incremental_checker.py

```diff
@@ -48,49 +61,56 @@ def execute(command: List[str], fail_on_error: bool = True) -> Tuple[str, str, i
     stdout_bytes, stderr_bytes = proc.communicate()  # type: Tuple[bytes, bytes]
     stdout, stderr = stdout_bytes.decode('utf-8'), stderr_bytes.decode('utf-8')
     if fail_on_error and proc.returncode != 0:
-        print("RETURN CODE:", proc.returncode)
+        print('EXECUTED COMMAND:', ' '.join(command))
+        print('RETURN CODE:', proc.returncode)
         print()
         print('STDOUT:')
         print_offset(stdout)
         print('STDERR:')
         print_offset(stderr)
-        raise Exception("Unexpected error from external tool.")
+        raise RuntimeError('Unexpected error from external tool.')
```

Comment:
It may be too late for this script, and mypy isn't consistent about this at all at all, but my personal favorite convention for when to use single vs. double quotes is to use double quotes for English text (e.g. error messages like this) and single quotes for stuff that's meaningful to some code (e.g. dict keys). You seem to be settling on the opposite convention? (Double quotes for English are handy because of things like "it's".)